cypress/integration/messaging/emoji_size_spec.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 5]*/
+
+function checkEmojiSize(message, emojis) {
+    emojis.forEach((emoji) => {
+        cy.get('@newLineMessage').
```

Comment:
I think you meant to use the `message` variable where you are using `@newLineMessage`.

Reply:
Yes. Thanks for catching that.